server/src/database/config/build.js

```diff
@@ -5,8 +5,12 @@ const connection = require('./connection');
 
 const dbBuild = () => {
   const dbFile = readFileSync(join(__dirname, 'build.sql')).toString();
-
-  return connection.query(dbFile);
+  const category = readFileSync(join(__dirname, 'category.sql')).toString();
+  const fakeData = readFileSync(join(__dirname, 'fakeData.sql')).toString();
+  return connection
+    .query(dbFile)
+    .then(connection.query(category))
```

Comment:
this will be executed immediately
without waiting  for the next promise
you can fix it like this
```suggestion
    .then(() => connection.query(category))
```